.github/advanced-issue-labeler.yml

```diff
@@ -67,7 +67,7 @@ policy:
             keys: ['Protectli VP4630', 'Protectli VP4650', 'Protectli VP4670']
 
           - name: protectli_vault_adl
-            keys: ['Protectli VP6630', 'Protectli VP6650', 'Protectli VP6670']
+            keys: ['Protectli VP6630', 'Protectli VP6650', 'Protectli VP6670', 'Protectli VP3210', 'Protectli VP3230']
```

Comment:
@SebastianCzapla The VP3000 series is much different from the VP6000 series. It is still Alder Lake, but the VP3000 is ADL-N. We likely need a new label in the same way as we have in the source code: `vault_adln`

Reply:
Fixed